src/Command/TemplateCommand.php

```diff
@@ -278,6 +285,7 @@ protected function _loadController(ConsoleIo $io): array
             $singularHumanName = $this->_singularHumanName($this->controllerName);
             $schema = $modelObject->getSchema();
             $fields = $schema->columns();
+            $hidden = $modelObject->newEmptyEntity()->getHidden() ?: ['token', 'password', 'passwd'];
```

Comment:
Should we have a constant for the list of fields hidden by default? That would prevent divergence in the future.

Reply:
I was just going to rely on the Entity class, but we don't have them in the test suite. Should we exclude anything if the user doesn't have an entity class defined yet?

Reply:
> Should we exclude anything if the user doesn't have an entity class defined yet?

You currently have the entity taking ownership of the hidden fields list which is :+1:. We can, and should provide reasonable fallbacks if the developer hasn't defined any fields as hidden.